README.md

```diff
@@ -17,12 +17,10 @@ Desktop Frontend of Qabel
    ```
 0. [start the servers](https://github.com/Qabel/qabel/blob/master/README.md#starting-the-servers) in a new terminal
 
-0. run (example with helloworld-module)
-
-   ```
-   java -Djava.library.path=../qabel-core -cp "../qabel-helloworld-module/build/libs/qabel-helloworld-module-0.1.jar:../qabel-desktop/build/libs/qabel-desktop-0.1.jar" de.qabel.desktop.QblMain -module qabel-helloworld-module/build/libs/qabel-helloworld-module-0.1.jar:de.qabel.helloworld.QblHelloWorldModule
-   ```
-
 ### all-in-one build
 
 travis uses the build.sh. it starts the required servers, runs the tests and stops the servers afterwards.
+
+## running Qabel Desktop
+
+A simple `./gradlew run` does not work. Use `./gradlew distZip` instead, unzip it and copy the curve library file (e.g. `qabel-core/build/binaries/curve25519SharedLibrary/osx_amd64/libcurve25519.dylib`) into the libs folder of the unzipped dist file prior to running the application.
```

Comment:
Actually, it **should** work.
a `gradlew build` also builds the qabel-core module, which builds the libcurve. `gradlew run` runs with the dependencies of both projects and the `java.library.path` set to the directory containing the libcurve. So even if it doesn't help, on *my system* it works.

I'm also wondering, why your distZip doesn't contain the libcurve. At the end of the build.gradle is the code, who copies the lib to the dist folder:
```
distZip() {
    baseName = project.name + '-' + platform

    from (configurations.compile) {
        into baseName + '-' + version + '/lib/'
    }
    from ("qabel-core/build/binaries/curve25519SharedLibrary/$platform/") {
        into baseName + '-' + version + '/lib/'
    }
}
```

I think we should better search the source of this problem than documenting it doesn't work (on macos).